src/request-pipeline/utils.ts

```diff
@@ -36,8 +37,14 @@ export function sendRequest (ctx: RequestPipelineContext) {
             resolve();
         });
 
-        req.on('error', () => {
-            ctx.hasDestReqErr = true;
+        req.on('error', err => {
+            // NOTE: Sometimes the underlying socket emits an error event. But if we have a response body,
+            // we can still process such requests. (B234324)
+            if (ctx.isDestResBodyMalformed()) {
+                error(ctx, getText(MESSAGE.destConnectionTerminated, ctx.dest.url, err.toString()));
```

Comment:
Need to come up with a more generic message, not only about connection terminating.
Something like this
```j
Failed to perform a request for the resource at <a href="{url}">{url}</a> because of error.\n{message}
```

Reply:
```
Failed to perform a request to the resource at <a href="{url}">{url}</a> because of an error.\n{message}
```